plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/glue/TrinoGlueCatalog.java

```diff
@@ -728,7 +729,7 @@ private void doCreateView(ConnectorSession session, SchemaTableName schemaViewNa
     {
         Optional<com.amazonaws.services.glue.model.Table> existing = getTableAndCacheMetadata(session, schemaViewName);
         if (existing.isPresent()) {
-            if (!replace || !isPrestoView(getTableParameters(existing.get()))) {
```

Comment:
In theory, the old code could replace an MV with a View.

Reply:
Could we add a test to verify it isn't allowed now ?

Reply:
same answer here: https://github.com/trinodb/trino/pull/18570#discussion_r1289743844